stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -1959,10 +1960,10 @@ pub mod test {
         key: &StacksPrivateKey,
         nonce: u64,
         signer_index: u128,
-        aggregate_public_key: &Point,
+        aggregate_public_key: &Secp256k1PublicKey,
```

Comment:
This is wrong.  The aggregate public key is a `Point`

Reply:
I reverted this change

Reply:
Sorry my mistake.

`test_signers.aggregate_public_key` is a `Point` too.